src/server/views/help/what-happens-next/submit-a-complaint-for-weeds-that-are-causing-damage-or-harm.html

```diff
@@ -1,6 +1,5 @@
 <h2 class="govuk-heading-m">What happens next</h2>
 
-<p class="govuk-body">The Defra Forms Service team aim to respond to you within 5 working days.</p>
 <p class="govuk-body">The Injurious Weeds Complaints Team at Natural England aim to respond to valid complaint forms and emails about ongoing open cases within 10 working days.</p>
```

Comment:
Just for consistency, have we used "Team" versus "team" elsewhere too?

```suggestion
<p class="govuk-body">The Injurious Weeds Complaints team at Natural England aim to respond to valid complaint forms and emails about ongoing open cases within 10 working days.</p>
```